RAGworkshop/README.md

```diff
@@ -0,0 +1,426 @@
+
+
+# Hands on! Using real time data in AI models
+
+
+## Overview
+
+This workshop will introduce realtime data as a key ally in developing AI models. We will cover realtime data structure, common patterns and common technologies used to hand realtime data. We will utilise the tools and techniques with which you are comfortable to explore realtime data so that it can be seamlessly integrated into your development workflow.
+
+The jewel of the session will be the development of a hierarchical condensation RAG based model utilising an open source realtime dataset to create a chatbot able to answer detailed questions around the current themes in the gaming community today.
+
+### Learning Outcomes
+
+By the end of this session you will:
+
+* Understand the core language and concepts of realtime data.  
+* Be able to discover, explore and prepare a realtime data source for RAG  
+* Use Python within a Jupyter notebook to enhance a prompt with realtime data
+
+## Preparation
+
+**Prerequisites**
```

Comment:
actually we should just get rid of pre-requisites altogether. We provide superset and jupyter and they should use our provided ones for everything. Please check we encourage this throughout the whole document.

Reply:
removed the prerequisite and mentioned only the the docker